scalableminds/webknossos
# Copy to my account for Volume Tracings

This has been available for skeletons for some time, now I added it for volumes, hopefully decreasing the pressure on #3007

### URL of deployed dev instance (used for testing):
- https://copyvolumetomyaccount.webknossos.xyz

### Steps to test:
- register second user, create volume tracing, trace some
- login as admin/scmboy, open the other user’s volume tracing via link
- click “copy to my account”, should create new explorative volume tracing in scmboy account, should contain data + description etc., but new timestamp

### Issues:
- fixes #2123

------
